get_all_preds.sh

```diff
@@ -0,0 +1,46 @@
+echo -n "blyr ... " && python tools/inference.py --path-model=models/d15.blyr.f.model --model-class=blyr --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "ceph ... " && python tools/inference.py --path-model=models/d15.ceph.f.model --model-class=ceph --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "dscu ... " && python tools/inference.py --path-model=models/d15.dscu.f.model --model-class=dscu --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "e ... "    && python tools/inference.py --path-model=models/d15.e.c.model --model-class=e --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "ea ... "   && python tools/inference.py --path-model=models/d15.ea.f.model --model-class=ea --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "eb ... "   && python tools/inference.py --path-model=models/d15.eb.f.model --model-class=eb --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "ew ... "   && python tools/inference.py --path-model=models/d15.ew.f.model --model-class=ew --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "fla ... "  && python tools/inference.py --path-model=models/d15.fla.c.model --model-class=fla --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "i ... "    && python tools/inference.py --path-model=models/d15.i.c.model --model-class=i --xgb-model --field=$1 --whole-field && echo "done"
+# echo -n "longt ... " && python tools/inference.py --path-model=models/d15.model --model-class=longt --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "lpv ... "  && python tools/inference.py --path-model=models/d15.lpv.f.model --model-class=lpv --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "pnp ... "  && python tools/inference.py --path-model=models/d15.pnp.c.model --model-class=pnp --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "puls ... " && python tools/inference.py --path-model=models/d15.puls.c.model --model-class=puls --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "rrlyr ... " && python tools/inference.py --path-model=models/d15.rrlyr.f.model --model-class=rrlyr --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "rscvn ... " && python tools/inference.py --path-model=models/d15.rscvn.f.model --model-class=rscvn --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "srv ... "  && python tools/inference.py --path-model=models/d15.srv.f.model --model-class=srv --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "vnv ... "  && python tools/inference.py --path-model=models/d15.vnv.c.model --model-class=vnv --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "wuma ... "  && python tools/inference.py --path-model=models/d15.wuma.f.model --model-class=wuma --xgb-model --field=$1 --whole-field && echo "done"
+echo -n "yso ... "  && python tools/inference.py --path-model=models/d15.yso.f.model --model-class=yso --xgb-model --field=$1 --whole-field && echo "done"
```

Comment:
Yes, these classes could be included in the script if needed (their trained models are also available). I did not include them because there were no available corresponding dnn models. Should I include these in the script now?

Reply:
Thanks, you can go ahead and include them. I think it's best to include all the available models, especially since we can comment out those lines if we don't want to use them in any given run.

Reply:
I agree. @saagar-parikh downstream we were requiring models that were implementted in xgboost and dnn, but that logic can stay separate for those kinds of specific requirements.